pkg/controller/identity/iamgrouppolicyattachment/controller.go

```diff
@@ -177,9 +159,8 @@ func (e *external) Create(ctx context.Context, mgd resource.Managed) (managed.Ex
 	// external identity. We therefore derive an external name from the
 	// names of the group and user that are bound.
 	meta.SetExternalName(cr, cr.Spec.ForProvider.GroupName+"/"+cr.Spec.ForProvider.PolicyARN)
->>>>>>> upstream/master
 
-	return managed.ExternalCreation{ExternalNameAssigned: true}, nil
+	return managed.ExternalCreation{}, nil
```

Comment:
Is removal of `ExternalNameAssigned: true` intentional?

Reply:
Yes, that field has been deprecated. Runtime now assumes the external name is assigned in all cases.